data-prepper-plugins/key-value-processor/src/test/java/org/opensearch/dataprepper/plugins/processor/keyvalue/KeyValueProcessorTests.java

```diff
@@ -466,6 +467,20 @@ void testTrueThreeInputsDuplicateValuesKvProcessor() {
         assertThatKeyEquals(parsed_message, "key1", expectedValue);
     }
 
+    @Test
+    void testTrueRemoveBracketsKvProcessor() {
+        when(mockConfig.getRemoveBrackets()).thenReturn(true);
+
+        final Record<Event> record = getMessage("key1=(value1)&key2=[value2]&key3=<value3>");
```

Comment:
Shall we add some more corner cases? e.g. what is the expected output for the following?

`key1=((value1)`, `key1=[value1][value2]`